NAMESPACE

```diff
@@ -19,6 +20,9 @@ export(subset_wuse)
 export(time_series_data)
 import(DT)
 import(ggplot2)
+import(maptools)
+import(rgeos)
```

Comment:
I'm going to merge this with the errors. I think I have a solution for travis in the hydroMaps package. I'll give it a shot. 

I also got stuck off-line most of last week because my computer broke, but finally seems to be working. Soooo frustrating during the long airport layovers.